pokemongo_bot/__init__.py

```diff
@@ -324,14 +329,12 @@ def _set_starting_position(self):
                         '[x] Last in-game location was set as: {}'.format(
                             self.position))
                     logger.log('')
-
+                    
+                    has_position = True
                     return
             except:
-                if not self.config.location:
-                    sys.exit(
-                        "No cached Location. Please specify initial location.")
-                else:
-                    pass
+                sys.exit(
+                    "No cached Location. Please specify initial location.")
 
     def _get_pos_by_name(self, location_name):
         # Check if the given location is already a coordinate.
```

Comment:
release_config.json.example → release_config.json
This should not be tracked.

Invitation sent, please join our team.